compose/fullnode/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3.7'
+services:
+  bitcoind:
+    image: lncm/bitcoind:0.18
+    entrypoint: bitcoind
+    volumes:
+      - /media/noma/bitcoind:/root/.bitcoin
```

Comment:
Why two?

Reply:
This way, `datadir` can be changed independently by setting a single path in `docker-compose.yml`